src/transaction/combine.ts

```diff
@@ -1,42 +1,80 @@
 import * as _ from 'lodash'
 import binary from 'ripple-binary-codec'
-import * as utils from './utils'
 import BigNumber from 'bignumber.js'
+import {ValidationError} from '../common/errors'
 import {decodeAccountID} from 'ripple-address-codec'
 import {validate} from '../common'
 import {computeBinaryTransactionHash} from '../common/hashes'
+import {JsonObject} from 'ripple-binary-codec/dist/types/serialized-type'
```

Comment:
I _think_ this is the wrong JsonObject.. This is the one in ripple-binary-codec. Think theres a different one in ripple-lib, `TransactionJson`

Reply:
Really, we should put TransactionJson in the binary codec, and then import it into ripple-lib.